include/Accessor/Accessor.h

```diff
@@ -0,0 +1,104 @@
+//=== Accessor.h - Accessor class definition ----*- C++ -*-===//
+//
+// Part of the OMP-UL Project, under the MIT License.
+// See libompx/LICENSE.txt for license information
+//
+//===-------------------------------------------------------------===//
+///
+/// \file
+/// This file contains the declaration of the Accessor class, which is
+/// used to wrap dynamically allocated regions pointers to provide 
+/// transparent data movement through OpenMP declare mapper
+///
+//===-------------------------------------------------------------===//
```

Comment:
Replace OMP-UL with libompx. 

We should also go with lower case folder names everywhere.